### ISSUE TITLE
INI values saving failed on 7.4

### ISSUE THREAD

**Issue by ruudboon** [shivammathur/setup-php#164]
**Describe the bug**
Php ini values are throwing an error: `/opt/hostedtoolcache/linux.sh: 209: /opt/hostedtoolcache/linux.sh: cannot create /etc/php/7.4/cli/php.ini: Permission denied`
This only happening under 7.4

**To Reproduce**
```yml
- name: Setup PHP
        uses: shivammathur/setup-php@v1
        with:
          php-version: ${{ matrix.php-versions }}
          ini-values: apc.enable_cli=on, session.save_path=/tmp
          tools: pecl
          extensions: mbstring, intl, json, imagick
```

See:
https://github.com/ruudboon/cphalcon/runs/408927933?check_suite_focus=true#step:7:25

**Comment by shivammathur** [comment]
@ruudboon Can you test using `shivammathur/setup-php@develop` if this issue is fixed.

**Comment by ruudboon** [comment]
Thnx. It's running: https://github.com/ruudboon/cphalcon/actions/runs/31403707

**Comment by ruudboon** [comment]
Thnx. Error is gone. https://github.com/ruudboon/cphalcon/runs/408993883?check_suite_focus=true#step:7:24

**Comment by shivammathur** [comment]
👍 will be part of next release `1.8.0`

**Comment by ruudboon** [comment]
Cool. I will leave it on develop for now and watch your release.

**Comment by shivammathur** [comment]
Merged and `1.8.0` is released, you can switch back.
